### PR TITLE
Upgraded golang version

### PR DESCRIPTION
**Pull request by aman-harness**

### CODE REVIEW

File: docker/docker/Dockerfile.linux.amd64 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/kaniko-project/executor:v1.6.0
+FROM gcr.io/kaniko-project/executor:v1.8.0
```
Comment (shubham149) [inline docker/docker/Dockerfile.linux.amd64:1]: @aman-harness Can you test whether updated kaniko works for gcr, ecr & dockerhub? Last time there was an issue in updated kaniko version due to which we have to revert the change.

Comment (aman-harness) [reply docker/docker/Dockerfile.linux.amd64]: [](url
Screenshot 2022-03-11 at 12 22 16 AM
)
I ahve tested it for all 3 of them and it was working fine

**Review by shubham149 — APPROVED** [review]
